package/MDAnalysis/lib/formats/include/readdcd.h

```diff
@@ -136,7 +136,7 @@ static int write_dcdheader(fio_fd fd, const char *remarks, int natoms,
  * Output: 0 on success, negative error code on failure.
  * Side effects: coordinates are written to the dcd file.
  */
-static int write_dcdstep(fio_fd fd, int curstep, int curframe, 
```

Comment:
I'm not really in favor of such changes in the c files. There doesn't seem to be a reason for it.